server.js

```diff
@@ -0,0 +1,18 @@
+var path = require('path');
```

Comment:
yep. will do. Still used to people not being on 4+, but I guess that's a safe assumption now.